src/dfx/src/commands/build.rs

```diff
@@ -24,6 +24,10 @@ pub struct CanisterBuildOpts {
     #[clap(long)]
     check: bool,
 
+    /// Disable default WASM shrink after build.
+    #[clap(long)]
+    no_shrink: bool,
+
```

Comment:
If we make this a per-canister option in dfx.json option instead of a `dfx build` option, then the developer won't have to remember to specify it every time, and it will work with `dfx deploy`.